hudi-aws/src/test/java/org/apache/hudi/aws/sync/ITTestGluePartitionPushdown.java

```diff
@@ -60,8 +60,9 @@
 
 @Disabled("HUDI-7475 The tests do not work. Disabling them to unblock Azure CI")
 public class ITTestGluePartitionPushdown {
-
-  private static final String MOTO_ENDPOINT = "http://localhost:5000";
+  // This port number must be the same as {@code moto.port} defined in pom.xml
+  private static final int MOTO_PORT = 5002;
+  private static final String MOTO_ENDPOINT = "http://localhost:" + MOTO_PORT;
```

Comment:
To make this integration test run locally.

Reply:
is this test even running currently? It's marked disabled.

Reply:
I run it locally to make sure the setup is still good.